build/Defaults/app.config

```diff
@@ -19,7 +18,51 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="System.Collections.Immutable" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
-        <bindingRedirect oldVersion="0.0.0.0-1.2.0.0" newVersion="1.2.0.0"/>
+        <bindingRedirect oldVersion="0.0.0.0-1.2.1.0" newVersion="1.2.1.0"/>
+      </dependentAssembly>
```

Comment:
It's not broken as long as binding redirects are in place. Many CoreFX 1.1 packages are built this way, not just SRM and SCI. @ericstj would be the right person to ask for details behind the decision.